data/matchingData/flatFare.json

```diff
@@ -4,6 +4,11 @@
   "type": "flatFare",
   "email": "test@example.com",
   "uuid": "BLACc1363964",
+  "timeRestriction": {
+    "startTime": "0900",
+    "endTime": "1500",
+    "validDays": ["monday", "tuesday", "wednesday", "thursday", "friday"]
+  },
```

Comment:
Some different examples would be good across these files, for example only one valid day, or only a start time etc.